plugins/PromisedScene/main.js

```diff
@@ -177,13 +186,47 @@ module.exports = async ({
             gettingStudio.push(JSON.parse(line).name);
           }
         }
-      });
 
+        if (!JSON.parse(line).aliases) {
+          return;
+        }
+
+        const allStudioAliases = JSON.parse(line).aliases.toString().split(",");
+
+        allStudioAliases.forEach((studioAlias) => {
+          if (studioAlias) {
+            let matchAliasStudio = new RegExp(studioAlias, "i");
+
+            let foundAliasStudioMatch = util.stripStr(scenePath).match(matchAliasStudio);
+
+            if (foundAliasStudioMatch !== null) {
+              gettingStudio.push("alias:" + JSON.parse(line).name);
+            } else {
+              const aliasNoSpaces = studioAlias.toString().replace(" ", "");
+
+              matchAliasStudio = new RegExp(aliasNoSpaces, "i");
+
+              foundAliasStudioMatch = util.stripStr(scenePath).match(matchAliasStudio);
+
+              if (foundAliasStudioMatch !== null) {
+                gettingStudio.push("alias:" + JSON.parse(line).name);
+              }
+            }
+          }
+        });
+      });
     // this is a debug option to se see how many studios were found by just doing a simple regex
     // $log(GettingStudio);
     let studiohighscore = 5000;
     if (gettingStudio.length && Array.isArray(gettingStudio)) {
+      let foundStudioAnAlias = 0;
       gettingStudio.forEach((stud) => {
+        let InstancefoundStudioAnAlias = 0;
```

Comment:
```suggestion
        let instanceFoundStudioAnAlias = false;
```
Change to camelCase as well